pkg/acceptance/helpers/account_client.go

```diff
@@ -92,6 +84,32 @@ func (c *AccountClient) Alter(t *testing.T, opts *sdk.AlterAccountOptions) {
 	require.NoError(t, err)
 }
 
+func (c *AccountClient) UnsetPoliciesFunc(t *testing.T) func() {
```

Comment:
this method is dangerous because it can alter the "wrong" account. We should at least have some safeguards (for now).

Reply:
As discussed SelfAlter tests for account are skipped. For now it's pointing to the general ticket (but mentioned it explicitly in the acc criteria).